server/src/grpc/storage_service/route.rs

```diff
@@ -13,11 +18,61 @@ pub async fn handle_route<Q>(
     ctx: &HandlerContext<'_, Q>,
     req: RouteRequest,
 ) -> Result<RouteResponse> {
+    // TODO: the case sensitive mode with quoted is not supported now, all table
```

Comment:
Maybe we should file a new issue to track the changes about the case sensitivity. As for this PR, maybe we need a configuration option called `case-insensitive = true` to enable such conversion. 

BTW, in the future changes, I am in favor of that CeresDB may be case-sensitive by default.